src/store_rebuild.h

```diff
@@ -12,10 +12,17 @@
 #define SQUID_STORE_REBUILD_H_
 
 #include "store_key_md5.h"
+#include <sys/time.h>
```

Comment:
Assuming the following comment applies to this file, let's remove this `#include`:

```
src/SquidTime.h:/* NP: sys/time.h is provided by libcompat */
```

```suggestion
```